node/src/accountant/mod.rs

```diff
@@ -3291,91 +3149,237 @@ mod tests {
+                        earning_wallet: earning_wallet_routing_1.clone(),
+                        service_rate: 42,
+                        byte_rate: 24,
+                    },
+                    RoutingServiceConsumed {
+                        earning_wallet: earning_wallet_routing_2.clone(),
+                        service_rate: 52,
+                        byte_rate: 33,
+                    },
+                ],
             })
             .unwrap();
 
-        System::current().stop_with_code(0);
+        System::current().stop();
         system.run();
-        let more_money_payable_parameters = more_money_payable_parameters_arc.lock().unwrap();
+        let more_money_payable_params = more_money_payable_params_arc.lock().unwrap();
         assert_eq!(
-            more_money_payable_parameters[0],
-            (now, make_wallet("booga"), (1 * 42) + (1234 * 24))
+            more_money_payable_params
+                .iter()
+                .map(|(timestamp, wallet, amount)| (timestamp, wallet, amount))
+                .collect::<Vec<_>>(),
+            vec![
+                (&timestamp, &earning_wallet_exit, &((1 * 120) + (1200 * 30))),
+                (
+                    &timestamp,
+                    &earning_wallet_routing_1,
+                    &((1 * 42) + (3456 * 24))
+                ),
+                (
+                    &timestamp,
+                    &earning_wallet_routing_2,
+                    &((1 * 52) + (3456 * 33))
+                )
+            ]
         );
-        TestLogHandler::new().exists_log_containing(&format!(
-            "DEBUG: Accountant: Accruing debt to wallet {} for consuming exit service 1234 bytes",
-            earning_wallet
+        let test_log_handler = TestLogHandler::new();
+        let expected_msg_id = Accountant::msg_id(timestamp);
+        test_log_handler.exists_log_containing(&format!(
+            "DEBUG: Accountant: MsgId {}: Accruing debt to {} for consuming 1200 exited bytes",
+            expected_msg_id, earning_wallet_exit
+        ));
+        test_log_handler.exists_log_containing(&format!(
+            "DEBUG: Accountant: MsgId {}: Accruing debt to {} for consuming 3456 routed bytes",
+            expected_msg_id, earning_wallet_routing_1
+        ));
+        test_log_handler.exists_log_containing(&format!(
+            "DEBUG: Accountant: MsgId {}: Accruing debt to {} for consuming 3456 routed bytes",
+            expected_msg_id, earning_wallet_routing_2
         ));
     }
 
-    #[test]
-    fn report_exit_service_consumed_message_is_received_for_our_consuming_wallet() {
-        init_test_logging();
-        let consuming_wallet = make_wallet("own consuming wallet");
-        let config = bc_from_ac_plus_wallets(
-            make_accountant_config_null(),
-            consuming_wallet.clone(),
-            make_wallet("own earning wallet"),
-        );
+    fn do_we_ignore_own_wallets_when_recording_consumed_services(
```

Comment:
This name...can we have something that's easier to understand in context? This name makes it sound like the return type should be boolean.